src/Earnings/SFA.DAS.Earnings/Application/Earnings/LearningDeliveryPeriodisedValuesBuilder.cs

```diff
@@ -0,0 +1,82 @@
+﻿using ESFA.DC.ILR.FundingService.FM36.FundingOutput.Model.Output;
+using SFA.DAS.SharedOuterApi.InnerApi.Responses.Earnings;
+
+namespace SFA.DAS.Earnings.Application.Earnings;
+
+public static class LearningDeliveryPeriodisedValuesBuilder
+{
+    public static LearningDeliveryPeriodisedValues BuildWithSameValues(string attributeName, decimal value)
+    {
+        return new LearningDeliveryPeriodisedValues
+        {
+            AttributeName = attributeName,
+            Period1 = value,
+            Period2 = value,
+            Period3 = value,
+            Period4 = value,
+            Period5 = value,
+            Period6 = value,
+            Period7 = value,
+            Period8 = value,
+            Period9 = value,
+            Period10 = value,
+            Period11 = value,
+            Period12 = value
+        };
+    }
+
+    public static LearningDeliveryPeriodisedValues BuildInstPerPeriodValues(Apprenticeship earningsApprenticeship, short academicYear)
+    {
+        var instalments = earningsApprenticeship.Episodes
```

Comment:
I don't want to pollute the calling code with multiple instalment select queries but I have moved it into a provider method in the builder to avoid duplication